src/components/date_picker/super_date_picker/super_date_picker.test.tsx

```diff
@@ -167,6 +167,31 @@ describe('EuiSuperDatePicker', () => {
       );
     });
 
+    it('invokes onFocus callbacks on the date popover buttons', () => {
+      const focusMock = jest.fn();
+      const component = mount(
+        <EuiSuperDatePicker
+          onTimeChange={noop}
+          showUpdateButton={false}
+          onFocus={focusMock}
+        />
+      );
+
+      component
+        .find('button[data-test-subj="superDatePickerShowDatesButton"]')
+        .simulate('click');
```

Comment:
This button should also be triggering the focus handler, so we should simulate focus on it as well
```suggestion
      component
        .find('button[data-test-subj="superDatePickerShowDatesButton"]')
        .simulate('click')
        .simulate('focus');
      expect(focusMock).toBeCalledTimes(1);
```